backend/benches/benchmark.rs

```diff
@@ -0,0 +1,139 @@
+use chrono::{DateTime, TimeDelta, Utc};
+use criterion::{criterion_group, criterion_main, Criterion};
+
+use rand::Rng;
+use scheduling_backend::data_model::graph::DiscreteGraph;
+use scheduling_backend::scheduling::scheduler::{GlobalSchedulerAlgorithm, SchedulerAlgorithm};
+use scheduling_backend::scheduling::{
+    scheduler::NaiveSchedulerAlgorithm, task_for_scheduler::TaskForScheduler,
+};
+
+use either::*;
+use protocol::{tasks::TaskId, time::Timespan};
+
+struct TaskFactory {
+    task_id: TaskId,
+}
+impl TaskFactory {
+    fn get_new_task_id(&mut self) -> TaskId {
+        let res = self.task_id;
+        self.task_id += 1;
+        res
+    }
+    pub fn new() -> Self {
+        TaskFactory { task_id: 0.into() }
+    }
+    pub fn make_tasks(
+        &mut self,
+        amount: usize,
+        start: DateTime<Utc>,
+        max_effect: f64,
+        max_duration: TimeDelta,
```

Comment:
We've used the Duration alias all over the project so please change it here aswell
```suggestion
        max_duration: Duration,
```